dashboard/lib/projects_list.rb

```diff
@@ -269,9 +269,10 @@ def get_project_row_data(project, channel_id, student = nil, with_library = fals
     # pull various fields out of the user and project records to populate
     # a data structure that can be used to populate a UI component displaying a
     # single library or a list of libraries.
-    def get_library_row_data(project, channel_id, user = nil)
+    def get_library_row_data(project, channel_id, section_name, user = nil)
       project_value = project[:value] ? JSON.parse(project[:value]) : {}
       {
+        section: section_name,
```

Comment:
nit: rename this key to `sectionName` to avoid confusion between our section objects and this property